feed-drupal.xml

```diff
@@ -14,8 +14,14 @@
     {% if post.tags contains 'drupal-planet' %}
       <item>
         <title>{{ post.title | xml_escape }}</title>
-        <description>{{ post.drupal_planet_summary | xml_escape | markdownify | xml_escape | replace: '!SITE_URL', site.url' }}
```

Comment:
@chrisarusso I'd leave this format as it is. As I recall it took quite a bit of trial and error to get this working properly. Unless it's breaking something, let's leave it.